jaggery/sachet/models.py

```diff
@@ -61,18 +57,19 @@ class Catalog(BaseModel):
         ),
     )
     latest_version = models.CharField(max_length=255)
```

Comment:
both latest_version and latest_expiry can be null
if null, it implies that there is no cache in the Sachet (Stores)

Reply:
The fields has been set to null true